src/ode_solver/runge_kutta_ode_solver.cpp

```diff
@@ -75,11 +75,11 @@ void RungeKuttaODESolver<dim,real,n_rk_stages,MeshType>::step_in_time (real dt,
     //assemble solution from stages
     for (int i = 0; i < n_rk_stages; ++i){
         if (pseudotime){
-            const double CFL = butcher_tableau->b(i) * dt;
+            const double CFL = butcher_tableau->get_b(i) * dt;
```

Comment:
`this->butcher_tableau` instead of `butcher_tableau`

Reply:
Thanks for pointing that out. Fixed in latest commit.